### PR TITLE
Local demo Starkli + Katana

### PR DESCRIPTION
**Pull request by dubzn**
Command `demo-local-cli` was added to the Makefile where the same actions as in the `/local.rs` script are performed, but through terminal commands using **Starkli** and **Katana**.

### CODE REVIEW

File: scripts/demo_local_cli.sh [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    1 \
+    500000000000000000 0 \
+    1 \
+    4295128740 0 0 \
+
+OWNER_T0_BALANCE_AF_SWAP=$(starkli call $TOKEN_0_ADDRESS balanceOf $KATANA_ACCOUNT_ADDRESS)
+OWNER_T1_BALANCE_AF_SWAP=$(starkli call $TOKEN_1_ADDRESS balanceOf $KATANA_ACCOUNT_ADDRESS)
+
+echo -e "$GREEN\n==> Balance after swap$RESET"
+echo -e "$GREEN      TYAS0: $OWNER_T0_BALANCE_AF_SWAP\n      TYAS1: $OWNER_T1_BALANCE_AF_SWAP$RESET"
```
Comment (rcatalan98) [inline scripts/demo_local_cli.sh:129]: EOF blank line missing

File: scripts/demo_local_cli.sh
Comment (rcatalan98) [inline scripts/demo_local_cli.sh:1]: Can we rename the file to `run_local_demo` or something similar? The script by itself should make sense outside of the Makefile

File: Makefile [hunk 2]
```diff
@@ -43,9 +46,8 @@ endif
 declare-testnet:
```
Comment (rcatalan98) [inline Makefile:46]: Leave a comment saying in the future we would need to add the NFTManager here as well

**Review by rcatalan98 — COMMENTED** [review]
Great work! Please take a look at the comments

**Comment by rcatalan98** [comment]
Take a look at the CI for some reason its failing

**Review by dpinones — APPROVED** [review]

**Review by uri-99 — APPROVED** [review]

**Pull request merged**